### PR TITLE
Add partition support

### PR DESCRIPTION
**Pull request by austinmroczek**
Adds partition support and tests.  New class for Partition and Device.  Bump version to 0.60

### CODE REVIEW

**Comment by austinmroczek** [comment]
This is not quite ready.  Need to move arm/disarm functions into Location so it can more easily handle the partition status updates.

**Comment by austinmroczek** [comment]
Still need to add ability to arm/disarm individual partitions.

File: CHANGE_LOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Change Log
```
Comment (uchagani) [inline CHANGE_LOG.md:1]: Changelog is traditionally one word for the file name.

File: setup.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 setup(
     name="total_connect_client",
     py_modules=["total_connect_client"],
-    version="0.58",
+    version="0.60",
```
Comment (uchagani) [inline setup.py:7]: Just FYI, home assistant devs will ask for what changed in between versions when you skip version numbers like this.

File: total_connect_client/TotalConnectClient.py [hunk 3]
```diff
@@ -322,55 +316,52 @@ def keep_alive(self):
 
     def arm_away(self, location_id):
         """Arm the system (Away)."""
+        # DEPRECATED
+        logging.debug("Using deprecated client.arm_away(). " "Use location.arm_away().")
```
Comment (uchagani) [inline total_connect_client/TotalConnectClient.py:320]: I would upgrade this logger to a warning for depreciations or use http://www.jaggedverge.com/2016/09/deprecation-warnings-in-python/

**Review by uchagani — APPROVED** [review]
Just a few suggestions. Thank you for keeping this project going

File: setup.py (hunk 2 above)
Comment (austinmroczek) [reply setup.py:7]: Yeah...hence the other issue/discussion about version numbers.  I was thinking such a large change was more than just 0.58 to 0.59...but also I wasn't confident enough to call it 1.0 or something like that.

File: CHANGE_LOG.md (hunk 1 above)
Comment (austinmroczek) [reply CHANGE_LOG.md:1]: done

File: total_connect_client/TotalConnectClient.py (hunk 3 above)
Comment (austinmroczek) [reply total_connect_client/TotalConnectClient.py]: will do

**Pull request merged**